doc/source/operations.rst

```diff
@@ -55,23 +55,60 @@ Once the appropriate profile has been selected, it can be applied by running the
 Updates
 =======
 
-Update Airship UCP Services
----------------------------
+SUSE Containerized OpenStack is delivered as an rpm package, so performing updates is generally accomplished by simply updating the rpm package to the latest version and redeploying the cloud using the steps outlined in the installation procedure. This is the typical update path and will incorporate all recent changes, as well as automatically updating component chart and image versions. However, there may be use cases where it is desirable to update services or components to specific versions. 
```

Comment:
is this reflected in a spec somewhere or it was agreed off-meeting?

Reply:
in the ccp build plan spec under cloud9 folder.